raiden/channel/balance_proof.py

```diff
@@ -209,30 +209,38 @@ def get_known_unlocks(self):
             self.hashlock_unlockedlocks.itervalues(),
         )
 
+        tree = self.generate_merkle_tree();
```

Comment:
As you can see the PR failed our style tests. Use `flake8` on all files like:

```
flake8 raiden/ tools/ --exclude */ansible/
```

to check for style incosistencies before submitting a PR.